core/tests/gae_suite.py

```diff
@@ -106,17 +94,12 @@ def _iterate(test_suite_or_case):
     results = [unittest.TextTestRunner(verbosity=2).run(suite)
                for suite in suites]
 
-    tests_run = 0
     for result in results:
-        tests_run += result.testsRun
         if result.errors or result.failures:
             raise Exception(
                 'Test suite failed: %s tests run, %s errors, %s failures.' % (
                     result.testsRun, len(result.errors), len(result.failures)))
 
-    if tests_run == 0:
-        raise Exception('No tests were run.')
```

Comment:
Why remove this?

Reply:
> Just curious, these seem to be 2 completely different files, how did you decide to write tests for them together?

Hi @nithusha21 there is no specific reason. I just wanted to cover these files together in a same PR.

Regarding the deletion, this gets checked in backend_tests.py [here](https://github.com/oppia/oppia/blob/624f6d9f2101147045d9cdc19f8e21699eccc57b/scripts/backend_tests.py#L306). Also create_test_suites() won't return an empty list --either a test that fails or that succeeds

Reply:
Ah ok.